python/ray/train/lightning/_lightning_utils.py

```diff
@@ -215,7 +215,25 @@ def prepare_trainer(trainer: pl.Trainer) -> pl.Trainer:
 
 @PublicAPI(stability="beta")
 class RayTrainReportCallback(pl.callbacks.Callback):
-    """A simple callback that reports checkpoints to Ray on train epoch end."""
+    """A simple callback that reports checkpoints to Ray on train epoch end.
+
+    This callback is a subclass of `lightning.pytorch.callbacks.Callback
+    <https://lightning.ai/docs/pytorch/stable/api/lightning.pytorch.callbacks.Callback.html#lightning.pytorch.callbacks.Callback>`_.
+
+    It fetches the latest `trainer.callback_metrics` and reports together with
+    the checkpoint on each training epoch end.
+
+    Checkpoints will be saved in the following structure:
+
+    .. testcode::
+
+        # checkpoint_00000*/      Ray Train Checkpoint
+        # └─ checkpoint.ckpt      PyTorch Lightning Checkpoint
```

Comment:
Can you try this:

```suggestion
    Checkpoints will be saved in the following structure::

        checkpoint_00000*/      Ray Train Checkpoint
        └─ checkpoint.ckpt      PyTorch Lightning Checkpoint
```

Can we do this instead:




https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#literal-blocks